bitnami/concourse/README.md

```diff
@@ -0,0 +1,398 @@
+| `externalDatabase.password`                  | Database password                                                                                            | `nil`               |
+| `externalDatabase.existingSecret`            | Name of an existing secret resource containing the DB password                                               | `nil`               |
+| `externalDatabase.existingSecretPasswordKey` | Name of an existing secret key containing the DB password                                                    | `nil`               |
+| `externalDatabase.database`                  | Database name                                                                                                | `bitnami_concourse` |
+| `externalDatabase.port`                      | Database port number                                                                                         | `5432`              |
+
+
+See https://github.com/bitnami-labs/readme-generator-for-helm to create the table
+
+The above parameters map to the env variables defined in [bitnami/concourse](http://github.com/bitnami/bitnami-docker-concourse). For more information please refer to the [bitnami/concourse](http://github.com/bitnami/bitnami-docker-concourse) image documentation.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+helm install my-release \
+  --set concourseUsername=admin \
+  --set concoursePassword=password \
+  --set mariadb.auth.rootPassword=secretpassword \
+    bitnami/concourse
+```
+
+The above command sets the concourse administrator account username and password to `admin` and `password` respectively. Additionally, it sets the MariaDB `root` user password to `secretpassword`.
+
+> NOTE: Once this chart is deployed, it is not possible to change the application's access credentials, such as usernames or passwords, using Helm. To change these application credentials after deployment, delete any persistent volumes (PVs) used by the chart and re-deploy it, or use the application's built-in administrative tools if available.
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+helm install my-release -f values.yaml bitnami/concourse
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Configuration and installation details
+
+### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)
+
+It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
+
+Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+### External database support
+
+%%IF NEEDED%%
```

Comment:
Template here. Also, check what other sections we have in charts (such as Ingress, TLS secrets, for example)